runatlantis.io/docs/server-configuration.md

```diff
@@ -342,6 +342,13 @@ Values are chosen in this order:
   ```
   Path to a GitHub App PEM encoded private key file. If set, GitHub authentication will be performed as [an installation](https://developer.github.com/v3/apps/installations/).
 
+- ### `--gh-app-key`
+  ```bash
+  export PRIVATE_KEY=$(cat path/to/app-key.pem)
+  atlantis server --gh-app-key=$PRIVATE_KEY
```

Comment:
If the file was available on disk, users should rather use `--gh-app-key-file`, let's make this example a little more concrete: 

```suggestion
  atlantis server --gh-app-key="-----BEGIN RSA PRIVATE KEY-----(...)" 
```